shell/platform/windows/text_input_plugin.cc

```diff
@@ -197,9 +197,7 @@ void TextInputPlugin::ComposeChangeHook(const std::u16string& text,
   std::string text_before_change = active_model_->GetText();
   TextRange composing_before_change = active_model_->composing_range();
   active_model_->AddText(text);
-  cursor_pos += active_model_->composing_range().start();
-  active_model_->UpdateComposingText(text);
-  active_model_->SetSelection(TextRange(cursor_pos, cursor_pos));
+  active_model_->UpdateComposingText(text, TextRange(cursor_pos, cursor_pos));
```

Comment:
Not really related to this change, but isn't the composing text being added twice here?

Reply:
You mean between `AddText` as well as `UpdateComposingText`? It sort of looks like it, though perhaps we're relying on a side effect of one or the other?